crates/forge/bin/cmd/test/mod.rs

```diff
@@ -376,6 +377,29 @@ impl TestArgs {
         }
 
         let remote_chain_id = runner.evm_opts.get_remote_chain_id().await;
+        let known_contracts = runner.known_contracts.clone();
+
+        // Set up trace identifiers.
+        let mut identifier = TraceIdentifiers::new().with_local(&known_contracts);
```

Comment:
I'd move these after spawning the runner thread for faster start up times

Reply:
yep, makes sense

addressed both comments in a88df74